testing/tests/ui/struct-expressions.stderr

```diff
@@ -0,0 +1,63 @@
+error: expected closing `}` of struct expression
+ --> <source attribute>:1:13
+       "{ ..Default::default(), other: 24 } }}"
+  --> tests/ui/struct-expressions.rs:18:14
+   |
+18 |     source = r#"{{ InnerCell { ..Default::default(), other: 24 } }}"#,
+   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: failed to parse template source
+ --> <source attribute>:1:20
+       "}thing"
+  --> tests/ui/struct-expressions.rs:24:21
+   |
+24 | #[template(source = r#"some{{ InnerCell { }}thing"#, ext = "txt")]
+   |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: expected closing `}` of struct expression
+ --> <source attribute>:1:17
+       "{ foo }}thing"
+  --> tests/ui/struct-expressions.rs:28:21
+   |
+28 | #[template(source = r#"some{{ InnerCell { foo }}thing"#, ext = "txt")]
+   |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: expected closing `}` of struct expression
+ --> <source attribute>:1:17
+       "{ foo: }}thing"
+  --> tests/ui/struct-expressions.rs:32:21
+   |
+32 | #[template(source = r#"some{{ InnerCell { foo: }}thing"#, ext = "txt")]
+   |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: failed to parse template source
+ --> <source attribute>:1:28
+       "}thing"
+  --> tests/ui/struct-expressions.rs:36:21
+   |
+36 | #[template(source = r#"some{{ InnerCell { foo: 32 }}thing"#, ext = "txt")]
+   |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: failed to parse template source
```

Comment:
Same. Although, there is no "working" test which checks if trailing comma is supported, please add one.